src/testing/tree_construction/parser.rs

```diff
@@ -296,7 +297,7 @@ fn test(i: Span) -> IResult<Span, TestSpec> {
 
             TestSpec {
                 position,
-                data: data.to_string(),
+                data: data.to_string().trim_matches(|c| c == '\n').to_string(),
```

Comment:
data should only trim one '\n', but trim_matches will trim all '\n', this should modify. tests16.dat and tests26.dat remain failed case is because this.

Reply:
+1